_posts/2022-04-11-nwaku-update.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: post
+name:  "What's new in `nwaku`?"
+title:  "What's new in `nwaku`?"
+date:   2022-04-11 10:00:00 +0200
+author: hanno
+published: true
+permalink: /nwaku-update-2022-04
+categories: research
+summary: A summary of recent developments in the Nim-based Waku v2 client and a preview of current and future focus areas.
+image: /assets/img/vac.png
+discuss:
+---
+
+## Background
+
+If you've been following our [research log](https://vac.dev/research-log/),
+you'll know that many things have happened in the world of Waku v2 since [our last general update](./2020-02-14-waku-update.md).
```

Comment:
I would consider this our most general update probably https://vac.dev/waku-v2-ethereum-coscup

Reply:
This link is incorrect, need to use the one present in the `permalink` metadata of the target post.